.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode
+out
+mill.*
```

Comment:
```suggestion
out
mill.*
```

Reply:
Nice suggestion, as adding .vscode to .gitignore is my personal coding style. A new suggestion commit is added.